src/Interpreters/InterpreterExplainQuery.cpp

```diff
@@ -174,7 +174,8 @@ struct QueryPlanSettings
             {"actions", query_plan_options.actions},
             {"indexes", query_plan_options.indexes},
             {"optimize", optimize},
-            {"json", json}
+            {"json", json},
+            {"sortmode", query_plan_options.sort_mode},
```

Comment:
I think it may be more reasonable to add a common setting which shows all the known data stream properties.
Right now it can be sort description + sort mode, distinct columns